src/Sfa.Das.ApprenticeshipInfoService.Api/Controllers/ProvidersController.cs

```diff
@@ -117,6 +127,54 @@ public void Head(long ukprn)
             Get(ukprn);
         }
 
+        /// <summary>
+        /// Get list of active apprenticeships for a given provider
+        /// </summary>
+        /// <param name="ukprn">UKPRN</param>
+        /// <returns>A list of active apprenticeships sorted by name alphabetically, then type, then level</returns>
+        [SwaggerResponse(HttpStatusCode.OK, "OK", typeof(IEnumerable<ProviderApprenticeship>))]
+        [SwaggerResponse(HttpStatusCode.InternalServerError)]
+        [SwaggerResponse(HttpStatusCode.BadRequest, BadUkprnMessage)]
+        [Route("providers/{ukprn}/active-apprenticeships", Name = "GetActiveApprenticeshipsByProvider")]
+        [ExceptionHandling]
+        public IEnumerable<ProviderApprenticeship> GetActiveApprenticeshipsByProvider(long ukprn)
```

Comment:
Discussion for all! 🎉 
Should we call them apprenticeship training instead?
Apprenticeship does mean a different thing outside our application and if its a public facing API maybe we want to be more careful.